client/quota_test.go

```diff
@@ -56,7 +57,12 @@ func (cs *clientSuite) TestEnsureQuotaGroup(c *check.C) {
 		"group-name": "foo",
 		"parent":     "bar",
 		"snaps":      []interface{}{"snap-a", "snap-b"},
-		"max-memory": float64(1001),
+		"constraints": map[string]interface{}{
+			// meh this is exactly '1001' in the JSON request, but unless we
+			// parse it as a json.Number into an int64, it will show up typed as
+			// a float64, so this is easier to check in the test
+			"memory": 1001.0,
```

Comment:
Yeah, this is annoying and sometimes confusing, I hit this on a couple of occasions.

Reply:
I fixed it with @mvo5's suggestion to decode with json.Number